### PR TITLE
Small suggestions - use markdown-file, redirect users to discord and promote discussion in discord rather

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![Envio_Grant_Program](https://github.com/Float-Capital/envio-grant-program/assets/102967165/d2e698a2-9799-4618-b33c-e09612fb9e3f)
 
-​​To commemorate the launch of our first stable release, we’re extending an invitation to all blockchain developers to apply to the **Envio "Build Bigger. Ship Faster" Grant Program**. The premise is simple yet incredibly rewarding: integrate a useful or interesting use case with Envio, share your invaluable feedback on the experience, and receive up to **$1000 USDC**.
+To commemorate the launch of our first stable alpha release, we’re extending an invitation to all blockchain developers to apply to the **Envio "Build Bigger. Ship Faster" Grant Program**. The premise is simple yet incredibly rewarding: integrate a useful or interesting use case with Envio, share your invaluable feedback on the experience, and receive up to **$1000 USDC**.
```
Comment (JasoonS) [inline README.md]: Very important they understand that it is alpha. Our goal with this is to get really good feedback from devs so we can iterate and improve the product. We aren't promising that the product is in some final form!

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Here are a few example ideas you can ship using Envio:
 - Applications are reviewed on a case-by-case basis. The more creative, complex, or impactful the project idea, the higher the grant amount you will receive.
 - Please allow for up to 2 working days (48 hours) for your project proposal to be reviewed.
 - Join our [Discord](https://discord.gg/mZHNWgNCAc) for updates and announcements on the “Build Bigger. Ship Faster” Grant Program, and introduce yourself to the team 👋
+    - This is alhpa software - hot of the press, so we are extremely appreciative of feedback - but more so we are there to give you a red carpet level of support if you have difficulties or questions.
```
Comment (JasoonS) [inline README.md:29]: We really need to drive home that users are testing and using alpha software. They will guaranteed run into difficulties that we need to smooth out. This is a big part of the motivation for the grants program in the first place. Also lots of dev discussion in discord looks really good to potential users! (I often have judged dev tools by the kinds of discord coversations happening)

File: project proposals/proposal_template.md [hunk 3]
```diff
@@ -19,7 +19,7 @@
 - [ ]  $250
 - [ ]  $100
 
-7. How much **time** do you think you need to build your Envio project? We will discuss and agree this timeline if you are application / proposal is accepted. 
+7. How much **time** do you think you need to build your Envio project? We will discuss and agree this timeline if your application / proposal is accepted.
```
Comment (JasoonS) [inline project proposals/proposal_template.md:22]: grammar

File: project proposals/proposal_template.md [hunk 4]
```diff
@@ -29,7 +29,10 @@
...
 - [ ]  Other: ______
 
-Anything not clear? Email us at hello@envio.dev or join our [Discord](https://discord.gg/mZHNWgNCAc), and we'll be happy to help and answer any questions. 
 
-Ship with us. By Builders. For Builders.
+**Anything not clear?** Join our [Discord](https://discord.gg/mZHNWgNCAc) where we have excelent support or if you prefer email us at [hello@envio.dev](mailto:hello@envio.dev). We'll be happy to help and answer any questions and discuss anything about the indexer.
+
+---
+
+**Ship with us. By Builders. For Builders.**
```
Comment (JasoonS) [inline project proposals/proposal_template.md:37]: Formatting and rewording. Focus should be on discord IMO.

File: project proposals/proposal_template.md
Comment (JasoonS) [inline project proposals/proposal_template.md:1]: Renamed this file - a `*.md` file is better.

**Review by SvenMuller95 — APPROVED** [review]

**Pull request merged**